### ISSUE TITLE
Bug: [`n/no-unsupported-features/node-builtins`] false-positive when writing to the variable

### ISSUE THREAD

**Issue by regseb** [eslint-community/eslint-plugin-n#274]
### Environment

Node version: v20.13.1
npm version: 10.5.2
ESLint version: 8.57.0
eslint-plugin-n version: 17.6.0
Operating System: Ubuntu 22.04.4 LTS

### What rule do you want to report?

n/no-unsupported-features/node-builtins

### Link to Minimal Reproducible Example

https://eslint-online-playground.netlify.app/#eNpNULtOxDAQ/JWVy9PFAQoQh+hoKKhAosAUTrwJ5py15Qe6U5R/x47Do1nLszOzszuz4PsWT3JyBvlnYAc2GttJ8/KhA3/F7tn2R4xwD72RIcDT+Rd7zBKYod0B5xx2LSx3gtieYTCaIu8tDXqslnpy1kcgGLydQGyUxpk0ampIsKwUhKeVpXCQyUR4EwQwlwJQmeGQ5xEs+wr6ZLBA9QfZl1qyTaKQXHFC1QwoY/IYMq6w6ZI2MdsIdighvLdesKpeypPLe93Byf4ox3IQSzn/OkKweHZYtZNVebhgaxLBFH49oENSSL3G1X9L9bNrld3yK36xqf56/+5QSJc3/LqwtkQLW74BNy2JMw==

### What did you expect to happen?

No error.

### Participation

- [ ] I am willing to submit a pull request for this issue.

### Additional comments

The error should be reported only when `WebSocket` is read, not when the `WebSocket` variable is written to.

In [my project](https://github.com/regseb/castkodi/blob/dd73d23dba3ece39d4dac53ac189aeb31e7701ba/test/polyfill.js#L17), I'm using the _mock-socket_ package as a `WebSocket` polyfill:

```javascript
import { WebSocket } from "mock-socket";

globalThis.WebSocket = WebSocket;
```

**Comment by scagood** [comment]
I am not sure I agree here, I think the rule should mark all references to unsupported features as errors.

Eg I always want `process.exitCode` to be marked as unsupported if it's not supported, despite the fact that I only write to it.

I would be inclined to say the correct course of action here is to use the `"ignores"` option.
